util/src/journaldb/earlymergedb.rs

```diff
@@ -290,7 +290,7 @@ impl HashDB for EarlyMergeDB {
 		ret
 	}
 
-	fn lookup(&self, key: &H256) -> Option<&[u8]> {
+	fn get(&self, key: &H256) -> Option<&[u8]> {
```

Comment:
i think this HashDB impl will fail to compile -- the deprecated methods are still required to implement the trait for backwards-compatibility reasons, and their replacements have default implementations based on those.